datastore/src/main/java/com/ds/datastore/BookStoreController.java

```diff
@@ -71,11 +76,17 @@ private void registerWithHub() throws Exception {
         JsonObject json = new JsonObject();
         json.addProperty("id", this.id);
         json.addProperty("address", this.url + "/bookstores/" + this.id);
-        outputJson(con, gson, json);
+        String str = gson.toJson(json);
```

Comment:
why are we not using `outputJson` any more?

Reply:
I think we reverted back for a second and we didn't catch this change